app/src/main/java/com/github/se/cyrcle/ui/profile/ProfileScreen.kt

```diff
@@ -0,0 +1,192 @@
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import coil.request.ImageRequest
+import coil.transform.CircleCropTransformation
+import com.github.se.cyrcle.model.user.User
+import com.github.se.cyrcle.ui.theme.ColorLevel
+import com.github.se.cyrcle.ui.theme.getButtonColors
+
+@Composable
+fun ProfileScreen(user: User) {
+  var isEditing by remember { mutableStateOf(false) }
+  var firstName by remember { mutableStateOf(TextFieldValue(user.firstName)) }
+  var lastName by remember { mutableStateOf(TextFieldValue(user.lastName)) }
+  var username by remember { mutableStateOf(TextFieldValue(user.username)) }
+
+  var favoriteParkings by remember { mutableStateOf(user.favoriteParkings.toMutableList()) }
+  val parkingFavorites = remember {
```

Comment:
It is hard to tell what this is and what it does. Is this just for demo purposes? I think if we click on the star, and therefore remove a parking from favorites, it should simply dissapear (opt: maybe we can add a confirmation card in a future sprint).
You don't need to "simulate" this feature while the viewmodel is not integrated. It seems like unnecesary overhead